src/components/DraftbotBreakdown.js

```diff
@@ -80,204 +80,158 @@ export const getPackAsSeen = (initialState, index, deck, seatIndex) => {
-    for (const c of picked.cards.WUBRG) {
-      probabilities[c.cardID] = getCastingProbability(c, lands);
-    }
-    const pickedInCombination = picked.cards.WUBRG.filter((c) => probabilities[c.cardID] > PROB_TO_INCLUDE);
+const WEIGHT_COLUMNS = Object.freeze([
+  { title: 'Oracle', sortable: true, key: 'title', heading: true, renderFn: renderWithTooltip },
+  { title: 'Weight', sortable: true, key: 'weight' },
+]);
 
-    for (let i = 0; i < TRAITS.length - 2; i++) {
-      card.scores.push(
-        TRAITS[i].function(colors, card, picked, draft.synergies, seen, lands, pickedInCombination, probabilities),
+export const Internal = ({ cardsInPack, draft, pack, picks, picked, seen }) => {
+  const [normalized, toggleNormalized] = useToggle(false);
+  const weights = useMemo(
+    () =>
+      TRAITS.filter((t) => t.weight).map(({ title, tooltip, weight }) => ({
+        title,
+        tooltip,
+        weight: weight(pack + 1, picks + 1, draft.initial_state),
+      })),
+    [draft, pack, picks],
+  );
+  const rows = useMemo(() => {
+    const res = cardsInPack.map((card) => {
+      const { rating, colors, lands } = botRatingAndCombination(
+        card,
+        picked,
+        seen,
+        draft.initial_state,
+        cardsInPack.length,
+        pack + 1,
       );
-    }
-    card.scores.push(JSON.stringify(fromEntries(Object.entries(lands).filter(([, c]) => c))));
-    card.scores.push(rating.toFixed(2));
-  }
-  if (normalized) {
-    for (let i = 0; i < TRAITS.length - 2; i++) {
-      let minScore = cardsInPack[0].scores[i];
-      let maxScore = cardsInPack[0].scores[i];
-      for (const card of cardsInPack) {
-        if (card.scores[i] < minScore) {
-          minScore = card.scores[i];
-        }
-        if (card.scores[i] > maxScore) {
-          maxScore = card.scores[i];
-        }
+      const probabilities = {};
+      for (const c of picked.cards.WUBRG) {
+        probabilities[c.cardID] = getCastingProbability(c, lands);
       }
-      for (const card of cardsInPack) {
-        card.scores[i] = `+${(card.scores[i] - minScore).toFixed(2)}`;
-      }
-    }
-  } else {
-    for (let i = 0; i < TRAITS.length - 2; i++) {
-      for (const card of cardsInPack) {
-        card.scores[i] = card.scores[i].toFixed(2);
-      }
-    }
-  }
-
-  const counts = useMemo(() => {
-    const res = [];
-
-    for (const card of cardsInPack) {
-      const row = { card };
-      for (let i = 0; i < card.scores.length; i++) {
-        row[TRAITS[i].name] = card.scores[i];
+      const pickedInCombination = picked.cards.WUBRG.filter((c) => probabilities[c.cardID] > PROB_TO_INCLUDE);
+      return fromEntries(
+        TRAITS.map(({ title, compute }) => [
+          title,
+          compute(colors, card, picked, draft.synergies, seen, lands, pickedInCombination, probabilities, rating),
+        ]),
+      );
+    });
+    if (normalized) {
+      for (const { title } of TRAITS) {
+        if (title !== 'Lands') {
```

Comment:
Card names also can't be normalized
```suggestion
        if (title !== 'Lands' && title !== 'Card') {
```